explorer/client/src/pages/object-result/views/TokenView.tsx

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import DisplayBox from '../../../components/displaybox/DisplayBox';
+import Longtext from '../../../components/longtext/Longtext';
+import ModulesWrapper from '../../../components/module/ModulesWrapper';
+import OwnedObjects from '../../../components/ownedobjects/OwnedObjects';
+import TxForID from '../../../components/transactions-for-id/TxForID';
+import { getOwnerStr, parseImageURL } from '../../../utils/objectUtils';
+import { trimStdLibPrefix } from '../../../utils/stringUtils';
+import { type DataType } from '../ObjectResultType';
+
+import styles from './ObjectView.module.css';
+function TokenView({ data, name }: { data: DataType; name?: string }) {
+    const viewedData = {
+        ...data,
+        objType: trimStdLibPrefix(data.objType),
+        name: data.name,
+        tx_digest: data.data.tx_digest,
+        owner: getOwnerStr(data.owner),
+        url: parseImageURL(data.data.contents),
+    };
+
+    const checkIsPropertyType = (value: any) =>
+        ['number', 'string'].includes(typeof value);
+    const stdLibRe = /0x2::/;
+    const prepObjTypeValue = (typeString: string) =>
+        typeString.replace(stdLibRe, '');
+
+    const properties = Object.entries(viewedData.data?.contents)
+        .filter(([key, _]) => key !== 'name')
```

Comment:
Why not do both filter checks in a single filter? i.e. `.filter(([key, value]) => key !== 'name' && checkIsPropertyType(value))`

Reply:
Good idea